components/CookieConsent.js

```diff
@@ -69,6 +74,13 @@ const pluginConfig = {
   },
 };
 
+const googleAnalyticsCookieScript = () => {
+  const cookie = JSON.parse(Cookies.get('cc_cookie'));
+  if (cookie?.categories.includes('necessary')) {
```

Comment:
I think this is not a necessary cookie. I think there should be a new section `analytics` or similar. Maybe @samyak-max could comment/help here, as I think there was such a section in his original implementation, before he removed it because there weren't any cookies being used.

Reply:
Yes, there was an analytics section. It can be created by adding a new obj in property blocks. Afaik this is an optional section so there must be a toggle.